doc/source/ray-air/predictors.rst

```diff
@@ -222,30 +220,6 @@ Before you begin
 
 .. tabs::
 
-    .. group-tab:: MXNet
-
-        First, install MXNet and Ray AIR.
-
-        .. code-block:: console
-
-            pip install mxnet 'ray[air]'
-
-        Then, import the objects required for this example.
-
-        .. literalinclude:: doc_code/mxnet_predictor.py
-            :language: python
-            :dedent:
-            :start-after: __mxnetpredictor_imports_start__
-            :end-before: __mxnetpredictor_imports_end__
-
-        Finally, create a stub for the `MXNetPredictor` class.
-
-        .. literalinclude:: doc_code/mxnet_predictor.py
-            :language: python
-            :dedent:
-            :start-after: __mxnetpredictor_signature_start__
-            :end-before: __mxnetpredictor_signature_end__
-
     .. group-tab:: statsmodel
```

Comment:
nit: Can remove tabs now since there's only one

Reply:
I left it in to make it easier to add more examples here in the future, but happy to remove :)